interfaces/mount/entry.go

```diff
@@ -141,3 +144,49 @@ func ParseEntry(s string) (Entry, error) {
 	e.CheckPassNumber = cpn
 	return e, nil
 }
+
+// LoadFSTab reads and parses an fstab-like file.
+//
+// The supported format is described by fstab(5).
+func LoadFSTab(reader io.Reader) ([]Entry, error) {
```

Comment:
(nitpick^2) - how about `LoadFstab/SaveFstab` ? Slightly easier to type and /etc/fstab is also fstab instead of FSTab?